### PR TITLE
remove file postprocessor/spectral_analysis/bijection.h

### LINKED ISSUE #76: Remove the file include/exadg/postprocessor/spectral_analysis/bijection.h
in reference to https://github.com/exadg/exadg/pull/74#discussion_r636043191

### PR DESCRIPTION
**Pull request by nfehn**
closes #76

### CODE REVIEW

**Review by kronbichler — APPROVED** [review]

**Comment by nfehn** [comment]
let's wait for another review by @peterrum

**Review by peterrum — APPROVED** [review]
Thanks.

**Pull request merged**
